CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!--## Unreleased-->
 
+## [2.0.0-alpha.40] - 2017-04-28
+
 * Added support for new JSDoc tags: @customElement, @polymer, @mixinFunction, @appliesMixin
 
 ## [2.0.0-alpha.39] - 2017-04-26
```

Comment:
These headers got messed up. Remove the duplicated bullet and the incorrect heading for alpha.38 below please.

The large list of features below was released in alpha 39 on the 26th